src/Infrastructure/Service/CommandRunner/CommandRunnerInterface.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Service\CommandRunner;
+
+use SprykerSdk\Sdk\Core\Domain\Entity\ContextInterface;
+use SprykerSdk\SdkContracts\Entity\CommandInterface;
+
+interface CommandRunnerInterface
```

Comment:
This one is used in the Application layer. Please move it to the `Application` layer